src/cascadia/TerminalApp/ContentManager.cpp

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "ContentManager.h"
+#include "ContentManager.g.cpp"
+
+#include <wil/token_helpers.h>
+
+#include "../../types/inc/utils.hpp"
+
+using namespace winrt::Windows::ApplicationModel;
+using namespace winrt::Windows::ApplicationModel::DataTransfer;
+using namespace winrt::Windows::UI::Xaml;
+using namespace winrt::Windows::UI::Xaml::Controls;
+using namespace winrt::Windows::UI::Core;
+using namespace winrt::Windows::System;
+using namespace winrt::Microsoft::Terminal;
+using namespace winrt::Microsoft::Terminal::Control;
+using namespace winrt::Microsoft::Terminal::Settings::Model;
+
+namespace winrt
+{
+    namespace MUX = Microsoft::UI::Xaml;
+    using IInspectable = Windows::Foundation::IInspectable;
+}
+namespace winrt::TerminalApp::implementation
+{
+    ControlInteractivity ContentManager::CreateCore(Microsoft::Terminal::Control::IControlSettings settings,
+                                                    IControlAppearance unfocusedAppearance,
+                                                    TerminalConnection::ITerminalConnection connection)
+    {
+        auto content = ControlInteractivity{ settings, unfocusedAppearance, connection };

```

Comment:
```suggestion
        ControlInteractivity content{ settings, unfocusedAppearance, connection };
```

Reply:
(I find certain uses of `auto` to be _less clear_ than the alternative, this one included. @lhecker?)

Reply:
I also prefer the suggestion. The rules around `auto` are much much more complex than they might appear initially. All of this and some more applies: https://eel.is/c++draft/temp.deduct
On the other hand declaring a concrete type like `ControlInteractivity` is very straightforward: https://eel.is/c++draft/class.ctor.general

---

As an aside, my personal coding style still hasn't changed lately. I still use `=` for assigning return values, because I find spotting `=` at a glance in a file, in particular with only basic syntax highlighting like in VS Code, much easier than spotting `{}`. It's also much closer to what every other programming language uses and so I feel more familiar. Finally, I find that it matches my mental model better, because even for the language itself, assigning variables and constructing objects are vastly different things. And so I use `=` when I'm basically giving temporary values a "name" (= assigning a variable name to the return value of a function) vs. constructing a brand new object on the stack.

When constructing new instances of a class or struct I always use the `Foo foo{ a, b, c };` style, because it's the one that has the least surprises from the language standard (as mentioned above).
But I'm also slightly biased against `{}` in general, because it's the exact same syntax that's used for initializer lists, and so you get these weird behaviors where this produces a vector of 2 items:
```cpp
std::vector<int> vec{ 10, 0 };
```
and this produces a vector of 10 items:
```cpp
std::vector<int> vec(10, 0);
```
The compiler will produce no warnings about either of those, and so we use `{}` for _everything_ in this project, except for `vector`s of course and I guess beginner C++ developers are just expected to know, always remember and to spot the difference between the two at all times? I can't call this good language design. Out of the infinite number of ASCII character combinations, it puzzles me why `{}` was chosen for both constructions of objects and initializer lists. You couldn't have maybe only used `{}` for initializer lists and simply make the rules around `()` stricter.....?
If I were to start a new C++ project myself, I'd probably use `()` with a good linter to spot narrowing (like clang-tidy, but I think VS can do it too).